bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
+
+#----install pigpio (npm install pigpio will be exec by tjbotlib package.json)
+curl abyz.co.uk/rpi/pigpio/pigpio.zip > pigpio.zip
+unzip -o pigpio.zip
+cd PIGPIO
+make
+sudo make install
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+
+npm install
+
+#----try to run tjbot
+if [ ! -f "${TJBOT_FOLDER}/recipes/conversation/config.js" ]; then
```

Comment:
this should probably run through the hardware tests in `bootstrap/tests` instead of running the recipe because this recipe needs watson credentials, which the user is highly likely not to have yet  :)